app/lnd/methods/index.js

```diff
@@ -42,13 +43,15 @@ export default function (lnd, meta, event, msg, data) {
     case 'getInvoiceAndQueryRoutes':
     // Data looks like { pubkey: String, amount: Number }
       invoicesController.getInvoice(lnd, meta, { pay_req: data.payreq })
-        .then((invoiceData) => {
-          networkController.queryRoutes(lnd, meta, { pubkey: invoiceData.destination, amount: invoiceData.num_satoshis })
-            .then((routes) => {
-              event.sender.send('receiveInvoiceAndQueryRoutes', routes)
-            })
-            .catch(error => console.log('getInvoiceAndQueryRoutes queryRoutes error: ', error))
-        })
+        .then(invoiceData =>
+          networkController.queryRoutes(lnd, meta, {
+            pubkey: invoiceData.destination,
+            amount: invoiceData.num_satoshis
```

Comment:
Incidentally I think we should adopt a practice of explicitly labelling variables by their units, ala `_satoshis`.

Reply:
yeah we need to come up with some unit standard going forward, especially because we want to allow unit switching within the app